tools/api_key_tool.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function
+import glob
+import re
+import argparse
+
+tokenString = '[Token '
+fixturesDirectory = 'tests/fixtures/'
+zeroapiregex = r'(\[Token )0{40}(\])'
+anyapiregex = r'(\[Token ).{40}(\])'
+zeroapistring = '[Token '+40*'0'+']'
+
+
+def api_key_detector(file):
```

Comment:
A convention for functions that return true/false is to name them `has_some_property`, could we name this method "has_api_key"?